### PR TITLE
Feature#34/settings functionality

### LINKED ISSUE #34: Settings_functionality
Add functionality for the settings view

### PR DESCRIPTION
**Pull request by Joshsam31**
# Description
- Added function to update data, added method to be able to update name, email, add description for user and change password.
- settings_controller and view were removed, and replaced by user_controller and editUser view, as they were not needed

# View
imagen

### CODE REVIEW

File: EasyLearn/app/views/users/editUser.html.erb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    <%= f.text_field :description, class: "form-control login_input" %>
+                  </div>
+                  <div class="form-label mb-3 row">
+                    <%= f.label :password, class: "d-flex justify-content-start login_label" %><br/>
+                    <%= f.password_field :password, class: "form-control login_input indata ml-3" %>
+                  </div>
+                  <div class="form-label mb-3 row">
+                    <%= f.submit "Save", class: "btn btn-dark login_button mb-3" %>
+					  	    </div>
+              <% end %>
```
Comment (YilKanda) [inline EasyLearn/app/views/users/editUser.html.erb]: What do you think about sending your form to an external file?

```suggestion
                    <%= render "users/form"%>
```

File: EasyLearn/app/views/users/editUser.html.erb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (YilKanda) [inline EasyLearn/app/views/users/editUser.html.erb]: Remove unnecessary spaces

File: EasyLearn/app/views/users/editUser.html.erb (hunk 1 above)
Comment (Joshsam31) [reply EasyLearn/app/views/users/editUser.html.erb]: done.

File: EasyLearn/app/views/users/editUser.html.erb (hunk 2 above)
Comment (Joshsam31) [reply EasyLearn/app/views/users/editUser.html.erb]: done.

File: EasyLearn/app/controllers/users_controller.rb [hunk 3]
```diff
@@ -15,8 +17,26 @@ def create
...
+  def user_params 
     params.require(:user).permit(:name, :email, :password)
   end
- 
+  def update_params 
+    params.require(:user).permit(:name, :email, :description ,:password)
+  end
+
+  
+
```
Comment (JuanequeX) [inline EasyLearn/app/controllers/users_controller.rb]: Delete this extra spaces

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: EasyLearn/app/controllers/users_controller.rb (hunk 3 above)
Comment (YilKanda) [reply EasyLearn/app/controllers/users_controller.rb]: I agree with @JuanequeX

**Review by YilKanda — APPROVED** [review]

**Review by BrandonVG — APPROVED** [review]

File: EasyLearn/app/controllers/users_controller.rb (hunk 3 above)
Comment (Joshsam31) [reply EasyLearn/app/controllers/users_controller.rb]: done.

**Pull request merged**